tests/mongodb_provision_cli.robot

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Documentation       Provision and Deploy MongoDB Database Instance from Developer View
+Metadata            Version    0.0.1
+
+Resource            ../resources/keywords/provision_dbinstance.resource
+
+Suite Setup         Set Library Search Order    SeleniumLibrary
+Suite Teardown      Tear Down The Test Suite
+Test Setup          Given Login To OpenShift CLI
+Test Teardown       Logout Of OpenShift CLI
```

Comment:
Should we leave the teardown as it is here? Or should we fix it now to fit the test suite scenarios (Like I did in my teardown PR)

Reply:
We could consolidate the changes after suite tear down and test tear down steps are incorporated into master branch.